### PR TITLE
Exchange rate emergency fix

### PR DESCRIPTION
**Pull request by matmair**
This PR switches the default exchange backend to frankfurter.app
Closes https://github.com/inventree/InvenTree/issues/5631

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
@matmair still failing some pricing related tests - any idea why?

Nice quick patch, very much appreciated :)

**Comment by wolflu05** [comment]
Maybe we should make this API url configurable, and default to frankfurter. That way ppl can host their own currency api server if they want to.

**Comment by SchrodingersGat** [comment]
> Maybe we should make this API url configurable, and default to frankfurter. That way ppl can host their own currency api server if they want to.

I like the idea of adding more options - as a future PR. The focus here should be on getting this merged in ASAP.

For future reference, I would like to see:

- Different backends supported (with potential for users to supply API keys if they have them)
- Allow exchange rates to be edited manually (no external updates)
- Potentially custom plugins can be used to fetch exchange rates

**Review by SchrodingersGat — APPROVED** [review]

**Comment by SchrodingersGat** [comment]
@matmair any chance to look at the errors here? Would be great to release this one ASAP

**Comment by SchrodingersGat** [comment]
@matmair very nice work. Are there any references in the documentation to the exchange rate system? If not, we should probably add something (in a separate PR)

**Comment by SchrodingersGat** [comment]
Backport to stable branch - https://github.com/inventree/InvenTree/pull/5650

**Comment by matmair** [comment]
@SchrodingersGat none that I am aware of.

File: InvenTree/InvenTree/exchange.py [hunk 1]
```diff
@@ -15,19 +15,19 @@
...
+    Uses the `frankfurter.app` service API
     """
 
-    name = "InvenTreeExchange"
+    name = "InvenTreeExchange-frankfurter"
 
     def __init__(self):
         """Set API url."""
-        self.url = "https://api.exchangerate.host/latest"
+        self.url = "https://api.frankfurter.app/latest"
```
Comment (Eolia) [inline InvenTree/InvenTree/exchange.py:24]: Error: spaces in url
Should be:
self.url = "https://api.frankfurter.app/latest"

**Pull request merged**